src/com/serotonin/mango/rt/dataImage/DataPointRT.java

```diff
@@ -82,10 +82,24 @@ public DataPointRT(DataPointVO vo, PointLocatorRT pointLocator) {
 		this.pointLocator = pointLocator;
 		valueCache = new PointValueCache(vo.getId(), vo.getDefaultCacheSize());
 	}
+	public DataPointRT(DataPointVO vo) {
+		this.vo = vo;
+		this.pointLocator = null;
+		valueCache = new PointValueCache();
+	}
+	public PointValueCache getPointValueCache(){
+		return this.valueCache;
+	}
 
 	public List<PointValueTime> getLatestPointValues(int limit) {
 		return valueCache.getLatestPointValues(limit);
 	}
+    public List<PointValueTime> getLatestPointValuesUsedForJunitTest(int limit) {
```

Comment:
I would prefer that there are no additional methods in the basic classes because we need some data in the tests.

Reply:
Grzegorz did you check what does actually this method or you see only text JunitTest. This method do not touch basic business logic of class and for me that is point that you don't do a review very carefully. Needed data (only needed) exists in jnuit tests.